### PR TITLE
Local file system storage as default

### PR DESCRIPTION
**Pull request by dennisreimann**
Checks whether or not a file storage has been set. If not, it sets the local file system storage as default.

### CODE REVIEW

File: BTCPayServer/Storage/StorageExtensions.cs [hunk 1]
```diff
@@ -25,49 +24,35 @@ public static void AddProviderStorage(this IServiceCollection serviceCollection)
...
-                {
-                    tmpdirInfo = Directory.CreateDirectory(datadirs.Value.TempStorageDir);
-                }
-                else
-                {
-                    tmpdirInfo = new DirectoryInfo(datadirs.Value.TempStorageDir);
-                }
+                var tmpdirInfo = Directory.Exists(datadirs.Value.TempStorageDir)
+                    ? new DirectoryInfo(datadirs.Value.TempStorageDir)
+                    : Directory.CreateDirectory(datadirs.Value.TempStorageDir);
```
Comment (dennisreimann) [inline BTCPayServer/Storage/StorageExtensions.cs:46]: These are just simplifications of the existing code.

**Comment by pavlenex** [comment]
Concept ack.

**Comment by NicolasDorier** [comment]
The PR is migrating current installs to use file storage provider.
However, it isn't putting the file storage provider for new instances. The migrations don't run for new installs.

**Comment by dennisreimann** [comment]
@NicolasDorier Updated as discussed.

**Review by dstrukt — APPROVED** [review]
tACK!

**Pull request merged**